README.md

```diff
@@ -99,6 +99,12 @@ CloudGatewayAddFilesystem -i <Filesystem Name> \
         <Instance Name>
 ```
 
+3. Create the cache folder
+```
+mkdir 0750 <Cache Directory Full Path>
```

Comment:
I'm pretty sure this command creates a directory named `0750` in the current working directory.

Reply:
should be good with the last commit